docs/src/validator/faq.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Validator Frequently Asked Questions
+sidebar_label: Frequently Asked Questions
+---
+
+### What is a validator?
+
+A validator is a computer that runs a software program to verify transactions that are added to the Solana blockchain.  A validator can be a voting validator or a non voting validator. To learn more, see [what is a validator](./overview/what-is-a-validator.md).
+
+### What is an RPC node?
+
+An RPC node is also a computer that runs the validator software.  Typically, an RPC node does not vote on the network.  Instead the RPC node's job is to respond to API requests.  See [what is an rpc node](./overview/what-is-an-rpc-node.md) for more information.
+
+### What is a cluster?
+
+For a definition and an overview of the topic, see [what is a cluster?](../cluster/overview.md). Solana maintains several clusters. For details on each, see [Solana clusters](../clusters.md).
+
+### What is Proof of Stake?
+
+Proof of Stake (PoS) is a blockchain architecture. Solana is a Proof of Stake blockchain. To read more, see [Proof of Stake](./overview/what-is-a-validator.md#proof-of-stake).
+
+### What is Proof of Work? Is running a Solana validator the same as mining?
+
+No, a Solana validator uses Proof of Stake. It does not use Proof of Work (often called mining). See [Proof of Work: For Contrast](./overview/what-is-a-validator.md#proof-of-stake).
+
+### Who can operate a validator?
```

Comment:
Maybe add a separate point addressing Validator Set (the fact there is no limited active validator set) as this is a stat people familiar with other blockchains may look for, as well as perhaps addressing different between block production and consensus participation (voting) and the associated fees (might be covered in the economic link already)

Reply:
Good point.  I added a separate question and answer and briefly mentioned the cost of voting.